wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,377 @@
+{% templatetag openblock %} load i18n {% templatetag closeblock %}
+<link rel="stylesheet" type="text/css" href="/static/admin/css/fonts.css">
+<style type="text/css">
+  body, main {
+    margin: 0 auto;
+  }
+  .body, .tip {
+    stroke: #fff;
+  }
+  html {
+    line-height: 1.15;
+    box-sizing: border-box;
+  }
+  footer, header, main {
+    display: block;
+  }
+  a {
+    background-color: transparent;
+    -webkit-text-decoration-skip: objects;
+    color: #246060;
+    text-decoration: none;
+  }
+  a:hover {
+    text-decoration: underline;
+  }
+  img {
+    border-style: none;
+  }
+  header {
+    border-bottom: 1px solid #efefef;
+  }
+  svg:not(:root) {
+    overflow: hidden;
+  }
+  body {
+    max-width: 960px;
+    color: #000000;
+    font-family: Open Sans, Arial, sans-serif;
+  }
+  main {
+    text-align: center;
+  }
+  h1, h2, h3, h4, h5, p, ul {
+    padding: 0;
+    margin: 0;
+    font-weight: 400;
+  }
+  header {
+    padding-top: 20px;
+    padding-bottom: 10px;
+  }
+  .row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .logo {
+    width: 200px;
+  }
+  .release-notes {
+    font-size: 14px;
+  }
+  .figure {
+    margin-top: 19vh;
+    height: 173px;
+  }
+  .figure-space {
+    max-width: 265px;
+    position: relative;
+    z-index: -9;
+  }
+  .figure-logo {
+    max-width: 150px;
+    position: relative;
+    z-index: -9;
```

Comment:
As far as I can see removing this `position` + `z-index` combo doesn't change anything to the rendered page? Same for `figure-space` above.

Reply:
Removed in 959d41b.